provision/minikube/cnpg/cluster.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cnpg-keycloak
+spec:
+  instances: 3
```

Comment:
Should we make this configurable, 3 is correct for multi-az ROSA deployments, but it seems overkill for minikube?

Reply:
Yes, I can add a param, e.g. `CNPG_INSTANCES`.

Reply:
I added the `CNPG_INSTANCES` parameter with a default value of `1`.